native/acter/src/api/push/notification_item.rs

```diff
@@ -900,12 +422,14 @@ impl NotificationItem {
         // acter specific items:
         if let RawNotificationEvent::Timeline(raw_tl) = &inner.raw_event {
             if let Ok(event) = raw_tl.deserialize_as::<SyncAnyActerEvent>() {
-                return NotificationItem::for_acter_object(
-                    client,
-                    builder,
-                    event.into_full_any_acter_event(room_id),
-                )
-                .await;
+                if !matches!(
```

Comment:
this actual if fixes a bug I only realized when looking at the code: as the `SyncAnyActerEvent` also catches regular matrix messages now, in the latest version of the code we _never_ go ahead to the `chat`-variant below. This match check actually ensures we skip those cases so we can do the chat message as below.